components/clp-py-utils/clp_py_utils/clp_config.py

```diff
@@ -597,7 +606,7 @@ def validate_port(cls, field):
 class CLPConfig(BaseModel):
     execution_container: Optional[str] = None
 
-    logs_input: Union[InputFsStorage, S3Storage] = InputFsStorage()
+    logs_input: Union[InputFsStorage, S3IngestionConfig] = InputFsStorage()
```

Comment:
if it's not a big change, can we also update `InputFsStorage` to `FsIngestionConfig`?

Note that I am aware of InputFsStorage is derived on "FsStorage". I think the naming of FsStorage is not accurate anymore for our current usage, but changing it will affect multiple place so let's not change it in this PR. keep FsStorage as it is for now.